package.json

```diff
@@ -19,15 +19,14 @@
     }
   },
   "dependencies": {
-    "@headlessui/react": "^1.7.14",
     "@emotion/react": "^11.11.0",
     "@emotion/styled": "^11.11.0",
     "@fortawesome/react-fontawesome": "^0.2.0",
+    "@headlessui/react": "^1.7.14",
     "@mui/icons-material": "^5.11.16",
     "@mui/material": "^5.13.2",
     "@mui/styled-engine-sc": "^5.12.0",
     "ci": "^2.2.0",
-    "husky": "^8.0.3",
```

Comment:
why did you remove `husky`?

Reply:
> why did you remove `husky`?

I didn't remove it maybe it is removed accidentally. I will solve this issue and  when I update my pull request I'll let u know. 🙂

Reply:
ok